pkg/internal/checkup/checkup.go

```diff
@@ -180,6 +187,36 @@ func (c *Checkup) Run(ctx context.Context) error {
 	return nil
 }
 
+func (c *Checkup) checkVersions(ctx context.Context, errStr *string) error {
+	log.Print("checkVersions")
+
+	ver, err := c.client.GetClusterVersion(ctx, "version")
+	if err != nil {
+		return err
+	}
+	ocpVersion := ""
+	for _, update := range ver.Status.History {
+		if update.State == configv1.CompletedUpdate {
+			// obtain the version from the last completed update
+			ocpVersion = update.Version
+			break
+		}
+	}
+
+	cdis, err := c.client.ListCDIs(ctx)
+	if err != nil {
+		return err
+	}
+	if len(cdis.Items) != 1 {
```

Comment:
maybe check if ==0 then have an error that it doesnt exist and if bigger then one then this current msg?